### PR TITLE
Adds sample config and jobscript file for compy

### LINKED ISSUE #598: Copy diagnostics and add example config/job script for CompyMcNodeFace
E3SM-Unified has been installed there.

To do:
- [x] copy public obs, mapping files and region masks
- [x] copy private obs, etc.
- [x] find an appropriate E3SM run on compy
- [x] add an example config file for the run
- [x] add and example job script

### PR DESCRIPTION
**Pull request by vanroekel**
adds compy config file and job script file.  

closes #598

### CODE REVIEW

File: configs/compy/config.20190711.testLuke.A_WCYCL1850S.ne30_oECv3.compy [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+#oceanStreamsFileName = streams.ocean
+#seaIceNamelistFileName = mpas-cice_in
+#seaIceStreamsFileName = streams.cice
+
+[output]
+## options related to writing out plots, intermediate cached data sets, logs,
+## etc.
+
+# directory where analysis should be written
+baseDirectory = /compyfs/vanr302/case_output/20190711.testLuke.A_WCYCL1850S.ne30_oECv3.compy_yr_30_40
```
Comment (xylar) [inline configs/compy/config.20190711.testLuke.A_WCYCL1850S.ne30_oECv3.compy]:
```suggestion
baseDirectory = /path/to/output/directory
```
We usually put nonsense here so people don't accidentally write output to your directory.

**Comment by xylar** [comment]
Small note: If you say "closes", "fixes" or "resolves" and then an issue number on its own lone, it gets closed automatically:
https://help.github.com/en/articles/closing-issues-using-keywords
"addresses" unfortunately doesn't work and I don't think it works if it's not on its own line.  I updated the description accordingly.

File: configs/compy/config.20190711.testLuke.A_WCYCL1850S.ne30_oECv3.compy (hunk 1 above)
Comment (vanroekel) [reply configs/compy/config.20190711.testLuke.A_WCYCL1850S.ne30_oECv3.compy]: Thanks, great point.  This is changed.

**Comment by vanroekel** [comment]
@xylar thanks for the suggestion on directory path and for the github tip.  I did not know that, obviously.

**Comment by xylar** [comment]
Thanks, @vanroekel!

**Pull request merged**